logbook-feign-logger/src/main/java/org/zalando/logbook/feignlogger/Localhost.java

```diff
@@ -0,0 +1,18 @@
+package org.zalando.logbook.feignlogger;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+interface Localhost {
```

Comment:
This interface was removed recently. See https://github.com/zalando/logbook/pull/201/files